src/mscorlib/src/System/Globalization/TextInfo.cs

```diff
@@ -684,6 +693,40 @@ private static bool IsLetterCategory(UnicodeCategory uc)
                  || uc == UnicodeCategory.OtherLetter);
         }
 
+
+        /// 
+        ///  Checks if the current culture is Dutch (i.e. "nl-BE" or "nl-NL") to
+        ///  determine if special casing rules should be applied. (Dutch Titlecasing)
+        ///
+        private bool IsDutchCulture(String str)
```

Comment:
use CultureName.StartsWith("nl-", StringComparison.OrdinalIgnoreCase) as we need to check all cultures starts with "nl-" 

using == will be case sensitive so if you have the name in the form "NL-NL"  for instance will not work